rllib/algorithms/impala/impala.py

```diff
@@ -109,7 +109,9 @@ def __init__(self, algo_class=None):
         self.vtrace = True
         self.vtrace_clip_rho_threshold = 1.0
         self.vtrace_clip_pg_rho_threshold = 1.0
-        self.vtrace_drop_last_ts = True
+        # TODO (sven): Deprecate this setting. It makes no sense to drop the last ts.
+        #  It's actually dangerous if there are important rewards "hiding" in that ts.
+        self.vtrace_drop_last_ts = False
```

Comment:
Flipped default value to False. This was already set to False in 99% of our tuned examples and learning tests.

Reply:
The APPO benchmarks on the new stack vs Atari also confirm that this setting either doesn't matter or is preferable over dropping the last ts.